framework/include/utils/NestedSolve.h

```diff
@@ -109,14 +113,22 @@ class NestedSolveTempl
   void
   nonlinearBounded(NSReal & guess, R & computeResidual, J & computeJacobian, B & computeBounds);
   ///@}
-
+  // @{ Perform a damped newton solve
+  template <typename R, typename J, typename C>
+  void nonlinearDamped(NSRealVectorValue & guess,
+                       R & computeResidual,
+                       J & computeJacobian,
+                       C & computeCondition);
+  ///@}
   ///@{ default values
   static Real relativeToleranceDefault() { return 1e-8; }
   static Real absoluteToleranceDefault() { return 1e-13; }
-  static Real xToleranceDefault() { return 1e-15; }
+  static Real xToleranceDefault() { return 1e-8; }
```

Comment:
why loosen this?

Reply:
Because it was originally an absolute step size tolerance, but is now a relative tolerance check (line 451 and 505).

Reply:
Tightened up the tolerance, I think 1e-15 makes more sense.